targets/TARGET_NORDIC/TARGET_NRF5x/TARGET_NRF52/TARGET_MCU_NRF52840/CMakeLists.txt

```diff
@@ -54,3 +57,11 @@ target_sources(mbed-core
 
         ${STARTUP_FILE}
 )
+
+mbed_set_linker_script(mbed-mcu-nrf52840 ${CMAKE_CURRENT_SOURCE_DIR}/${LINKER_FILE})
+
+target_link_libraries(mbed-mcu-nrf52840 INTERFACE mbed-nrf52 mbed-sdk-15-0)
+target_link_libraries(mbed-arduino-nano33ble INTERFACE mbed-mcu-nrf52840)
+target_link_libraries(mbed-ep-agora INTERFACE mbed-mcu-nrf52840)
+target_link_libraries(mbed-ep-atlas INTERFACE mbed-mcu-nrf52840) # TODO ask HK about this
```

Comment:
What did you want to ask here?

Reply:
I was having an issue with a dependency in mbed-cellular (similar to what we saw the last time) but found that the error goes away as long as you don't add the file ONBOARD_TELIT_ME910.cpp as a source.